### PR TITLE
Completed html-tables exercise. - Charlie McCanna

### PR DESCRIPTION
**Pull request by cgmccanna**
html-tables

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>Student Name</th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
```
Comment (BrettAlbright) [inline html-tables/index.html:15]: make sure to include the `tbody` element also

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by cgmccanna** [comment]
updated-html-tables

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**